policyengine_us/variables/gov/states/mt/tax/income/mt_taxable_social_security.py

```diff
@@ -0,0 +1,51 @@
+from policyengine_us.model_api import *
+
+
+class mt_taxable_social_security(Variable):
+    value_type = float
+    entity = TaxUnit 
+    label = "Montana taxable social security benefits"
+    defined_for = StateCode.MT
+    definition_period = YEAR
+    reference = "https://mtrevenue.gov/wp-content/uploads/mdocs/form%202%202021.pdf#page=6"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.mt.tax.income.social_security
+        # line 1&2: 6a in tax form ?
+        social_security_benefit = tax_unit("social_security",period)
+        fraction_of_social_security_benefit= social_security_benefit*p.rate1
+        # line 3
+        tax_unit("total_income",period) - tax_unit("taxable_social_security",period)
+        # line 4: Additions Schedule line 15 - line 3 
+        0
+        # line 5: 
+        tax_emempt_interest_income = tax_unit("tax_exempt_interest_income",period) 
+        # line 6: line 2+3+4+5
+        line_6 = fraction_of_social_security_benefit + tax_unit("total_income",period) - tax_unit("taxable_social_security",period) +tax_emempt_interest_income
```

Comment:
should it be capped at 0?

Reply:
The taxable_social_security is a part of the toatal_income, so it's always greater than 0.

Reply:
I see, could we split out the individual variables?